Dockerfile

```
@@ -1,4 +1,4 @@
-FROM pulumi/pulumi:v2.23.2
+FROM pulumi/pulumi:v2.24.0
```

```suggestion
FROM pulumi/pulumi:v2.24.1
```

Should skip v2.24.0 due to reverted bugs in that release.